.github/workflows/docker-slurm.yml

```diff
@@ -43,6 +43,12 @@ jobs:
           cd docker/spack-stack
           cat mirrors.in.yaml | envsubst > mirrors.yaml
           ./create_dockerfile.sh
+      -
+        name: Make lint
+        run:  |
+          pip install flake8
+          flake8
```

Comment:
Do you want to explicitly specify the `src` and `tests` directories for `flake8`?

e.g. `flake8 src tests` or something like that?

Reply:
Ok.  Let's leave it for now.  My concern is that if someone is doing some debugging and has accidentally left, for example, a spack-stack directory or something lying around it could start analyzing code that's not ours and take a really long time to run.  That's kind of an outlier though. I like having things simpler when possible.  And `flake8` is simpler than `flake8 src tests`.